draft-ietf-gnap-core-protocol.md

```diff
@@ -1288,6 +1295,9 @@ given at runtime. Additional fields sent during a request but not present in a p
 client instance record at the AS SHOULD NOT be added to the client's pre-registered record.
 See additional considerations regarding client instance impersonation in {{security-impersonation}}.
 
+A client instance that is capable of talking to multiple AS's SHOULD use a different key for each
```

Comment:
```suggestion
A client instance that is capable of talking to multiple ASs SHOULD use a different key for each
```
I'm not sure if the plural of `AS` should be `ASs` or `AS's`. `ASs` is used twice in the document, while `AS's` is also used twice.

Reply:
I don't think there's a good way to do it, to be honest, but I'm not a fan of "ASs" because it looks like a typo of "ass". Still you're right that the doc should be consistent, so we'll want to fix that eventually -- but I'm leaving these all as-is for now.